test/SmokeTests/BackendSmokeTests/Netherite/Dockerfile

```
@@ -10,7 +10,11 @@ RUN cd /root/test/SmokeTests/BackendSmokeTests/Netherite && \
     dotnet publish -c Release --output /home/site/wwwroot
 
 # Deploy the app
-FROM mcr.microsoft.com/azure-functions/dotnet:4.0
+# TODO: For some reason, the 'floating tag' `dotnet:4.0` does not contain
+# the latest Host. Our Netherite tests will fail without an up to date host.
+# Therefore, we're pinning to a more up to date tag than 4.0, but we should
+# eventually move to a 'floating tag' again.
+FROM mcr.microsoft.com/azure-functions/dotnet:4.22.0
```

@bachuv: I'm told the following edit could to get the "True" latest Docker image. Might trying it out? If it fails, you can revert back.

```suggestion
FROM mcr.microsoft.com/azure-functions/dotnet:4
```

---

Yes, I'll merge this change right now.

---

The Netherite smoke test passed with the change!